.github/workflows/release-please.yml

```diff
@@ -27,6 +27,7 @@ jobs:
           token: ${{ secrets.AZTEC_BOT_GITHUB_TOKEN }}
           command: manifest
           default-branch: ${{ env.BRANCH_NAME }}
+          release-type: node
```

Comment:
should this be `simple`? https://github.com/AztecProtocol/aztec-packages/blob/aedeabab1504e6c203e8771379b4d18fad0a3e73/release-please-config.json#L20-L21

Reply:
actually I don't know what the default is, this was not in master so I'll remove it